src/content/LandingPage/_landing-page.scss

```diff
@@ -0,0 +1,45 @@
+@import './_mixins.scss';
+@import './overrides.scss';
+@import 'carbon-components/scss/globals/scss/typography.scss';
+@import 'carbon-components/scss/globals/scss/vars.scss';
+
```

Comment:
You are missing the style here regarding .landing_page_illo.  Look in the tutorial for that section and make sure it is pasted here.